code/modules/admin/verbs/possess.dm

```diff
@@ -2,10 +2,9 @@
 	set name = "Possess Obj"
 	set category = "Object"
 
-	if(istype(O,/obj/singularity))
-		if(config.forbid_singulo_possession)
-			to_chat(usr, "It is forbidden to possess singularities.")
-			return
+	if(!O.dangerous_possession && config.forbid_singulo_possession)

```

Comment:
this will make you fail to possess every single item, except the ones you are trying to prevent